packages/design-system/src/styles/components/_Table.scss

```diff
@@ -55,37 +61,60 @@
     word-wrap: normal;
   }
 
-  th,
-  td {
+  tr,
+  caption {
+    display: block;
+  }
+
+  td,
+  th {
     border: 0;
+    display: grid;
+    grid-template-columns: 42% auto;
+    position: relative;
+    z-index: 100;
```

Comment:
what is this used for? is this to position it on top of the :before pseudo element?

Reply:
This is to position the `:before` element which is the table heading, on top of the `:after` element which is the grey background.

Reply:
could we style the after element to have -9999 z index instead?

Reply:
Updated to have  -9999 z index for the `:after` pseudo element